.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: An issue to track an existing bug in PartiQL
+title: ''
+labels: bug
+assignees: ''
```

Comment:
Is there any auto-update features from Github to link the fields with the columns on the right of the issue pages?

Reply:
Not sure what you mean. Can you elaborate? BTW these fields are pre-filled attributes. AKA, when someone creates an issue using the Bug Report template, it'll add the `bug` label. But, they can add more labels or remove the `bug` label in the UI.